tests/framework-tests/preferences/js/SelfVoicingPanelTests.js

```diff
@@ -62,6 +63,9 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
                 expect: 4,
                 name: "Test the rendering of the speak panel",
                 sequence: [{
+                    event: "{testEnvironment speak}.events.onResourcesFetched",
+                    listeners: "fluid.identity"
+                },  {
```

Comment:
I don't think this would add much. If it doesn't fire, the tests will hang, which is exactly what they will do now.